frontend/packages/core/src/Input/select.tsx

```diff
@@ -300,4 +300,128 @@ export const Select = ({
   );
 };
 
-export default Select;
+export interface MultiSelectProps extends Pick<MuiSelectProps, "disabled" | "error"> {
+  defaultOptions?: Array<number> | Array<string>;
+  helperText?: string;
+  label?: string;
+  name: string;
+  selectOptions: SelectOption[];
+  onChange?: (values: Array<string>) => void;
+  flex?: boolean;
+}
+
+const MultiSelect = ({
+  defaultOptions = [],
+  disabled,
+  error,
+  helperText,
+  label,
+  name,
+  selectOptions,
+  onChange,
+  flex = false,
+}: MultiSelectProps) => {
+  // Flattens all options and sub grouped options for easier retrieval
+  const flatOptions: BaseSelectOptions[] = flatten(
+    selectOptions.map((option: SelectOption) =>
+      option.group ? option.group.map(groupOption => groupOption) : option
+    )
+  );
+
+  // Will take a string or an integer and attempt to find the index where it exists based on the flattened items
+  const calculateDefaultOption = (): Array<number> => {
```

Comment:
Im tempted to say we pull this out into a helper function as well. The biggest difference is that it takes an array input which would be easy enough to support by casting it beforehand.

Reply:
going to try it out